### PR TITLE
bm.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Bm.ru.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!--realestate. refused-->
+<ruleset name="Bm.ru">
+	<target host="bm.ru" />
+	<target host="www.bm.ru" />
+	<target host="online.bm.ru" />
+	<target host="anketa.bm.ru" />
+	<target host="anketa-ipoteka.bm.ru" />
+	<target host="m.bm.ru" />
+	<target host="bmmobile.bm.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/Bm.ru.xml]: Please see also:
mobile.bm.ru
partner.bm.ru

File: src/chrome/content/rules/Bm.ru.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	
+	<exclusion pattern="^http://(?:www\.)?bm.ru/(?!ru/personal/platezhi-i-perevody/perevody-s-kart/perevody-s-karty-na-karty)" />
```
Comment (J0WI) [inline src/chrome/content/rules/Bm.ru.xml]: Please avoid non-capturing group and document the issue.

File: src/chrome/content/rules/Bm.ru.xml (hunk 1 above)
Comment (galeksandrp) [inline src/chrome/content/rules/Bm.ru.xml]: Added.

File: src/chrome/content/rules/Bm.ru.xml (hunk 2 above)
Comment (galeksandrp) [inline src/chrome/content/rules/Bm.ru.xml]: Done.

**Comment by J0WI** [comment]
I think you need to make an exception for them on `/$` to make our tests happy.

**Comment by galeksandrp** [comment]
No luck.

**Comment by galeksandrp** [comment]
Added.

**Pull request merged**